app/api/auth.py

```diff
@@ -345,6 +348,32 @@ def order_invoices(order_identifier):
         return ForbiddenError({'source': ''}, 'Authentication Required to access Invoice').respond()
 
 
+@ticket_blueprint.route('/events/invoices/<string:invoice_identifier>')
+@jwt_required()
+def event_invoices(invoice_identifier):
+    if current_user:
+        try:
+            event_invoice = EventInvoice.query.filter_by(identifier=invoice_identifier).first()
```

Comment:
Can we merge this and line below as 
`.first().event_id`

as NoResultFound will be be raise in any case. Let me know if that may not work or you think it will be a bad idea.

Reply:
@poush This will throw an `AttributeError` as the SQLAlchemy attempts to find `event_id` on the `NoneType` object. Ergo, I wouldn't recommend merging the two :)